src/test.ml

```diff
@@ -61,7 +61,7 @@ let test = Type.[
      [named_nullable "delta" Int];
   tt "select test.name from test where test.id + @x = ? or test.id - @x = ?"
      [attr' ~nullability:(Nullable) "name" Text;]
-     [named_nullable "x" Int; param_nullable Int; named_nullable "x" Int; param_nullable Int;];
+     [named_nullable "x" Int; param Int; named_nullable "x" Int; param Int;];
```

Comment:
this should be improved - x should be non-nullable too. ie not-nullable constraint should flow back through the expression somehow. I guess can save this improvement for another PR